### DISCUSSION TITLE
CoreDNS Mysql configuration

### DISCUSSION THREAD

**Discussion by iqzirr** [coredns/coredns#5090] [Q&A] [answered]
sorry for my bad english hheehe
i'm new in CoreDNS, and i want to try MySQL plugin to automatically fill my new deployment's records with MySQL client, I run this CoreDNS as my local DNS server, this is my Corefile config:
```
.:53 {
    mysql {
        dsn coredns:Corednstest@tcp(192.168.123.51:3306)/mydb
        table_prefix coredns_
        zone_update_interval 5
    }
    log
    errors
    forward . 8.8.8.8 {
        except test.com
        policy sequential
    }
}
```
and this is my data in DB
![image](https://user-images.githubusercontent.com/70002495/149281726-cc2d45e3-cb46-483b-bc7a-60b18f349f82.png)

can u guys plz check my config if I configured it wrong

**Comment by chrisohaver** [comment, marked as answer]
Since the mysql plugin is an external plugin, not maintained by coredns, you might have better chance of getting assistance from the maintainers of the mysql plugin at https://github.com/cloud66-oss/coredns_mysql .
